access_control.py

```diff
@@ -158,16 +173,108 @@ def _process_access_control_requests(
     return successful, failed
 
 
-def process_access_control_requests() -> Tuple[List[str], List[str], List[str]]:
+def process_access_control_requests() -> (
+    Tuple[List[str], List[str], List[str]]
+):
     """Process access control requests from both 'grant_access' and
     'revoke_access' directories.
     """
     print("Processing access control request")
-    granted, failed = _process_access_control_requests("grant_access", remove=False)
-    revoked, failed_2 = _process_access_control_requests("revoke_access", remove=True)
+    granted, failed = _process_access_control_requests(
+        "grant_access", remove=False
+    )
+    revoked, failed_2 = _process_access_control_requests(
+        "revoke_access", remove=True
+    )
     return granted, revoked, failed + failed_2
 
 
+def send_pr_cirun(
+    feedstock: str,
+    feedstock_dir: str,
+    resource: str,
+    cirun_policy_args: Optional[List[str]] = None,
+) -> None:
+    """
+    Send PR to feedstock to enable Github Actions with Cirun
+
+    Parameters:
+    feedstock (str): The name of the feedstock.
+    feedstock_dir (str): Path to a git checkout of the feedstock.
+    resource (str): The name of the resource for access control.
+    cirun_policy_args (List[str]): A list of policy arguments for cirun
+                                   resources.
+    """
+
+    with update_conda_forge_config(
+        os.path.join(feedstock_dir, "recipe", "conda_build_config.yaml")
+    ) as cbc, update_conda_forge_config(
+        os.path.join(feedstock_dir, "conda-forge.yml")
+    ) as cfg:
+        if any(
+            label.startwith("cirun-")
+            for label in cbc.get("github_actions_labels", [])
+        ):
+            return
+        cfg["github_actions"] = {"self_hosted: true"}
+        if "pull_request" in cirun_policy_args:
+            cfg["github_actions"]["triggers"] = ["push", "pull_request"]
+        if "provider" not in cfg:
+            cfg["provider"] = {}
+        cfg["provider"]["linux_64"] = "github_actions"
+        if "upload_on_branch" not in cfg:
+            cfg["upload_on_branch"] = ["main"]
+        cbc["github_actions_labels"] = [resource]
+
+    gh = Github(gh_token())
+    user = gh.get_user()
+
+    repo = gh.get_repo(f"{GH_ORG}/{feedstock}-feedstock")
+    repo.create_fork()
+
+    base_branch = f"cirun-{int(time.time())}"
+
+    git_cmds = [
+        ["git", "add", "recipe/conda_build_config.yaml", "conda-forge.yml"],
+        [
+            "git",
+            "remote",
+            "add",
+            user.login,
+            "https://x-access-token:${GITHUB_TOKEN}@github.com/"
+            f"{user.login}/{feedstock}.git",
+        ],
+        [
+            "git",
+            "commit",
+            "-m",
+            f"Enable {resource} using Cirun",
+            "--author",
+            f"{user.name} <{user.email}>",
+        ],
+        ["git", "push", user.login, f"HEAD:cirun-{base_branch}"],
```

Comment:
```suggestion
        ["git", "push", user.login, f"HEAD:{base_branch}"],
```

I think we already have the `cirun-` prefix in `base-branch`, don't we?